popup.js

```diff
@@ -0,0 +1,282 @@
+    <ul class="card-sub">
+      <li class="remove">${firstData[2].caption1}</li>
+      <li>${firstData[2].caption2}</li>
+      <li>${firstData[2].caption3}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${firstData[1].content}
+      </p>
+      <ul class="category">
+        <li>${firstData[2].technology[0]}</li>
+        <li>${firstData[2].technology[1]}</li>
+        <li>${firstData[2].technology[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+
+  <div class="card-body">
+  <div class="card-img swap">
+  <img src="${firstData[3].cardImg}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="card-header">${firstData[3].header}</h2>
+    <ul class="card-sub">
+      <li class="remove">${firstData[1].caption1}</li>
+      <li>${firstData[3].caption2}</li>
+      <li>${firstData[3].caption3}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${firstData[3].content}
+      </p>
+      <ul class="category">
+        <li>${firstData[3].technology[0]}</li>
+        <li>${firstData[3].technology[1]}</li>
+        <li>${firstData[3].technology[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+  `;
+
+for (let i = 0; i < showButton.length; i += 1) {
+  showButton[i].addEventListener('click', () => {
+    body.style.display = 'block';
+    document.body.scrollTop = 0;
+    document.documentElement.scrollTop = 0;
+
+    popUp.innerHTML = `
+        <section id="model-section">
+        <div class="model-body">
+          <div class="model-title">
+            <h2 class="card-header">${cardData[i].header}</h2>
+            <i id="model-close" class="uil uil-multiply"></i>
+            <ul class="card-sub">
+              <li class="remove">${cardData[i].caption1}</li>
+              <li>${cardData[i].caption2}</li>
+              <li>${cardData[i].caption3}</li>
+            </ul>
+          </div>
+          <img class="image-img" src="${cardData[i].cardImg}" width="100%"/>       
+          <div>
+            <div class="model-content">
+              <p>${cardData[i].content}</p>
+              <div class="model-links-content">
+                <ul class="category mobile-cat">
+                  <li>${cardData[i].technology[0]}</li>
+                  <li>${cardData[i].technology[1]}</li>
+                  <li>${cardData[i].technology[2]}</li>
+                </ul>
+                <div>
+                  <hr class="line-break" />
+                  <div class="model-combine">
+                    <button class="model-button live">
+                      See live
+                      <svg
+                        class="svg"
+                        width="18"
+                        height="18"
+                        viewBox="0 0 18 18"
+                        fill="none"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/232185178-a44edbb8-f89f-45f6-9d91-0bcc9e92e52e.png)


Figma
![image](https://user-images.githubusercontent.com/78251927/232185121-6d795935-ed41-41d7-a845-023d227c2b06.png)

Good job implementing the previous reviewer's suggestions 💯. However, I noticed that there is a recent issue in the `desktop-design` popup window we need to Fix 👀. so that our design will match the given template on Figma. 👍

Kindly adjust your styling so that the layout of the content of the `button` looks similar to the given template 🛠💪